integrationtest/wildfly/README.md

```diff
@@ -0,0 +1,8 @@
+## Explanation of testing profiles
+### Obtaining WildFly distribution
+- The default is that the TS obtains WildFly from Maven and prepares it automatically.
+- To test against WildFly distributions which you prepared yourself, just pass the `-DjbossHome.node1` and `-DjbossHome.node2` properties containing paths to the distributions
+
+### Testing included modules vs. preparing modules 
+- The default mode is that the TS unzips Hibernate Search and required libraries as WildFly modules into the distributions
+- If you want to test against the modules included in the official WildFly distributions, pass the property `-DuseBuiltinModules`
```

Comment:
I know we don't do this consistently, but maybe using qualified names for properties would be interesting here? Be it only to make it clear, from the property name, what it will affect?
I'd use something like `test.wildfly.node1.home`, `test.wildfly.node2.home` and `test.wildfly.useBuiltinModules`. Or maybe `it.wildfly.node1.home`, `it.wildfly.node2.home` and `it.wildfly.useBuiltinModules`, if you want to clearly separate integration tests from the rest.